cli/cli.py

```diff
@@ -0,0 +1,155 @@
+import os
+from linchpin_api.v1.api import LinchpinAPI
+from linchpin_api.v1.invoke_playbooks import invoke_linchpin
+from utils import parse_yaml
+
+
+class LinchpinCli(LinchpinAPI):
+
+    def __init__(self):
+        LinchpinAPI.__init__(self)
+
+    def test(self):
+        print("test function inside linchpincli")
+
+    def lp_topo_list(self, upstream=None):
+        return LinchpinAPI.lp_topo_list(self, upstream)
+
+    def lp_layout_list(self, upstream=None):
+        return LinchpinAPI.lp_layout_list(self, upstream)
+
+    def find_topology(self, topology, lpf):
+        config = self.get_config()
+        print("Finding topology in local folder ./topology")
+        topos = os.listdir("./topologies")
+        if topology in topos:
+            return os.path.abspath("./topologies/"+topology)
+        print("Not found")
```

Comment:
Consistency is important here. The user thinks you are looking in ./topology, but you actually look in ./topologies. Which is it?